### PR TITLE
libselinux: add v3.5, fix license info

### PR DESCRIPTION
**Pull request by valgur**
Really just to confirm that the `shared=True` build works ok in isolation. 
`libselinux` seems to be causing issues in the shared build for `cairo` in #18883.

### CODE REVIEW

**Review by RubenRBS — DISMISSED** [review]

**Comment by RubenRBS** [comment]
Thanks for investigating :)

**Review by RubenRBS — DISMISSED** [review]

**Review by ericLemanissier — CHANGES_REQUESTED** [review]
file 0001-fix-fno-common-3.0.patch needs to be restored

**Review by ericLemanissier — APPROVED** [review]

**Review by danimtb — DISMISSED** [review]

File: recipes/libselinux/all/conanfile.py [hunk 1]
```diff
@@ -20,7 +21,14 @@ class LibSELinuxConan(ConanFile):
...
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/SELinuxProject/selinux"
-    license = "Unlicense"
+    license = (
+        # https://github.com/SELinuxProject/selinux/blob/main/libselinux/LICENSE
+        # For the libselinux component: public domain with a limited liability clause
+        "LicenseRef-LICENSE-libselinux",
+        # https://github.com/SELinuxProject/selinux/blob/main/libsepol/LICENSE
+        # For the libsepol component: LGPL-2.1
+        "LGPL-2.1",
```
[the remark this answers is not in the document]
Comment (valgur) [reply recipes/libselinux/all/conanfile.py:30]: Good catch. Thanks!

**Review by jwillikers — APPROVED** [review]
Thanks! Appreciate these license corrections.

**Pull request merged**